File: baekjoon/greedy/14659/14659.java
import java.util.*;
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.BufferedWriter;
import java.io.OutputStreamWriter;

public class Main{

  public static void main(String []args) throws Exception{
    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

    int N = Integer.parseInt(br.readLine());
    String[] tmp = br.readLine().split(" ");
    int[] arrow = new int[N];
    int result = 0, cnt = 0, comp;

    for(int i=0;i<N;i++){
      arrow[i] = Integer.parseInt(tmp[i]);
    }

    comp = arrow[0];
    for(int i=1;i<N;i++){
      if(comp > arrow[i]) ++cnt;
      else{
        if(result < cnt) result = cnt;
        cnt = 0;
        comp = arrow[i];
      }
    }

    if(result < cnt) result = cnt;


    bw.write(""+result);    
    bw.flush();
    bw.close();
    br.close();
  } 
}

File: baekjoon/greedy/14659/14659.py
import sys

N = int(sys.stdin.readline())
arrow = list(map(int,sys.stdin.readline().split()))
result, cnt = 0, 0
comp = arrow[0]


for i in range(1,N):
  if comp > arrow[i]: cnt += 1
  else:
    result = max(result, cnt)
    comp = arrow[i]
    cnt = 0

result = max(result, cnt)
print(result)
